### PR TITLE
Create logitech-g304.device

### PR DESCRIPTION
**Pull request by BaconCondensate**
Add support to Logitech G304 (Asian market version of G305)

### CODE REVIEW

File: data/devices/logitech-g304.device [hunk 1]
```diff
@@ -0,0 +1,9 @@
+[Device]
+Name=Logitech Gaming Mouse G304
+DeviceMatch=usb:046d:c53f
```
Comment (staticssleever668) [inline data/devices/logitech-g304.device:3]: 046d:c53f is receiver's ID, which doesn't usually work with libratbag. Not sure why tests don't fail for that. 
Perhaps `lsusb` shows another Logitech device?
If not, try connecting it in wired mode.

File: data/devices/logitech-g304.device [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+Driver=hidpp20
+DeviceType=mouse
+
+[Driver/hidpp20]
+Leds=0
+Quirk=G304
```
Comment (staticssleever668) [inline data/devices/logitech-g304.device:9]: 'G304' is not a valid quirk, it should could either be 'G305' or not set at all. Our tests don't catch that right now, which I opened an issue for: #1557.

**Review by staticssleever668 — CHANGES_REQUESTED** [review]
Hello! Thank you for your PR, but it needs some changes

File: data/devices/logitech-g304.device (hunk 2 above)
Comment (BaconCondensate) [reply data/devices/logitech-g304.device:9]: Yeah, I'll need to change it to G305 then. It's my first time trying to do this so pardon my skill issue

**Comment by BaconCondensate** [comment]
After a lot of testing, I have concluded that G304 have the exact same PID as G305, and the reason it was not detected is not due to the lack of device file, but due to a bug on my end. Closing MR

**Pull request closed without merging**